package.json

```diff
@@ -24,7 +24,6 @@
     "@radix-ui/react-tooltip": "^0.1.6",
```

Comment:
any reason why you have made these updates? - had issues when starting the dev server as react was missing.

Don't think we need to include this file in the PR :)

Reply:
@srindom oh shoot, I was trying to clean things up late at night. let me fix this.